### PR TITLE
Minevale cb melone edit

### PR DESCRIPTION
**Pull request by Plattio**
## Type of change

- [ ] Add new directory for a server
- [x ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [ x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [ x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x ] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/minevale/manifest.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+  "server_name": "minevale",
+  "nice_name": "MineVale",
+  "direct_ip": "minevale.de",
+  "server_wildcards": [
+    "%.minevale.de",
```
Comment (maaaathis) [inline minecraft_servers/minevale/manifest.json:6]: Doesn't look like a real wildcard:
image

File: minecraft_servers/minevale/manifest.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    "%.minevale.eu"
```
Comment (maaaathis) [inline minecraft_servers/minevale/manifest.json:7]: Doesn't look like a real wildcard:
image

File: minecraft_servers/minevale/manifest.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      "color": "#52ff59",
+      "command": "/queue citybuild2"
+    },
+    "citybuild3": {
+      "name": "CityBuild 3",
+      "color": "#52ff59",
+      "command": "/queue citybuild3"
+    },
+    "citybuild4": {
+      "name": "CityBuild Melone",
```
Comment (maaaathis) [inline minecraft_servers/minevale/manifest.json:39]: key doesn't match to name

**Review by maaaathis — CHANGES_REQUESTED** [review]

**Comment by DoJapHD** [comment]
Inactive PR

**Pull request closed without merging**